lib/pages/survey/survey_controller.dart

```diff
@@ -10,6 +10,17 @@ class SurveyController extends GetxController {
 
   Optional<Survey> get optionalSurvey => _survey.value;
 
+  int get questionAmount => optionalSurvey.value.questions.length;
+
+  final _currentQuestion = 0.obs;
+
+  int get currentQuestionIndex => _currentQuestion.value;
+
+  SurveyQuestion get currentQuestion =>
+      optionalSurvey.value.questions[currentQuestionIndex];
+
+  String get indexString => _getIndexString();
```

Comment:
perhaps `questionPositionString` 💭

Reply:
maybe `indexTitleText` could be a clearer name 🤔

Reply:
that works for me 🙆‍♂️